source/interfaces/q2studio.rst

```diff
@@ -7,6 +7,15 @@ QIIME 2 Studio (``q2studio``) is a graphical user interface for QIIME 2. This do
 
 .. note:: QIIME 2 Studio currently requires a source installation. In the future, we will provide a package distribution/installer which will greatly simplify the installation process presented here. An experimental build of QIIME 2 Studio is available as a pre-installed package in the :doc:`VirtualBox image <../install/virtual/virtualbox>`.
 
+Before being able to successfully run QIIME 2 Studio, you must always confirm that your QIIME 2 ``conda`` environment is activated. Failing to activate your environment prior to starting QIIME 2 Studio will cause issues and most likely crash the application. If you have not activated your ``conda`` environment already, you should do so now:
```

Comment:
Nice! I have a few suggested tweaks, but overall I like the flow!

```
Launching QIIME 2 Studio requires an active ``conda`` environment
that has QIIME 2 installed in it. Failure to activate a compatible
``conda`` environment will likely produce errors, and possibly
cause the application to crash. If you have not activated your ``conda``
environment already, please do so now:
```

Feel free to tighten that up a bit if something jumps out at you!